src/player.js

```diff
@@ -130,53 +158,95 @@ class Player extends FakeEventTarget {
    * @static
    */
   static _defaultConfig(): Object {
-    return {};
+    return DefaultPlayerConfig;
```

Comment:
Make this immutable, we don't want default config to change.
you can just serialize it via object.assign